embed/config.go

```diff
@@ -246,7 +248,7 @@ func (cfg *configYAML) configFromFile(path string) error {
 		cfg.ACUrls = []url.URL(u)
 	}
 
-	if (cfg.Durl != "" || cfg.DNSCluster != "") && cfg.InitialCluster == cfg.InitialClusterFromName(cfg.Name) {
+	if (cfg.Durl != "" || cfg.DNSCluster != "") && cfg.InitialCluster == defaultInitialCluster {
```

Comment:
Done... although it's tricky to explain :-\

Not a fan of all the default values set by `NewConfig`, as it makes for more complicated undo/override code like this.